README.md

```diff
@@ -823,6 +823,20 @@ client.createApplication(bookmarkAppOptions).then((createdApp: Application) => {
 
 ## Migrating between versions
 
+### From 4.x to 5.0
+
+#### Breaking changes
+
+Following methods signatures have changed:
```

Comment:
Let's be specific if the changes happen in `client` or `model`, if too much information is needed here, feel free to show some information here and link to the changelog, looks like we have a very detailed list there.